ml/embeddings.py

```diff
@@ -138,7 +138,7 @@ def fit(self, graph: Union[nx.DiGraph, np.ndarray], weight='length'):
             else:
                 raise ValueError(f'Invalid weight transform: {self._weight_transform}')
 
-        A = nx.to_scipy_sparse_matrix(graph, nodelist=sorted(graph.nodes),
+        A = nx.convert_matrix.to_scipy_sparse_array(graph, nodelist=sorted(graph.nodes),
```

Comment:
`Cannot find reference 'to_scipy_sparse_matrix' in '__init__.py' `

I can revert it, but I always change it locally because it doesn't work without it

Reply:
Do you know your version of networkx library? Could you please share it here?